src/osd/osd_types.h

```diff
@@ -4604,14 +4608,20 @@ struct object_manifest_t {
     TYPE_NONE = 0,
     TYPE_REDIRECT = 1, 
     TYPE_CHUNKED = 2, 
+    TYPE_DEDUPED = 3, 
```

Comment:
This doesn't seem like the place to add it.  REDIRECT means the whole object is elsewhere, CHUNKED means pieces of it are elsewhere.  Dedup seems like a particular case of *either*.  Can we stick with CHUNKED here?  Some chunks maybe deduped chunks, some not; some may be refcouned, some not.  The current types already allow us to express that.

Reply:
Right. Thanks for your comment.